app/connectors/CustomsDeclareExportsMovementsConnector.scala

```diff
@@ -62,4 +64,42 @@ class CustomsDeclareExportsMovementsConnector @Inject()(appConfig: AppConfig, ht
 
   private def logFailedExchange(`type`: String, exception: Throwable): Unit =
     logger.warn(`type` + " failed", exception)
+
+  def fetchAllSubmissions(providerId: String)(implicit hc: HeaderCarrier): Future[Seq[SubmissionFrontendModel]] =
```

Comment:
I think this service calls it `pid` rather than `providerId`, it'd be good to be consistent otherwise we'll look back in the future and forget they're the same thing

Reply:
The whole backend uses `providerId` so I am going to stay with this here.

Reply:
We should be consistent either way and change the `pid`s

Reply:
Changed everything to `providerId`